proto/viam/app/v1/app.proto

```diff
@@ -680,47 +681,38 @@ message DeleteRobotPartSecretRequest {
 message DeleteRobotPartSecretResponse {}
 
 message Authorization {
-  string authorization_type = 1;
-  string authorization_id = 2;
-  string resource_type = 3;
-  string resource_id = 4;
+  string identity_id = 1; 
+  string authorization_type = 2;
+  string authorization_id = 3;
+  string resource_type = 4;
+  string resource_id = 5;
+  string org_id = 6;
 }
 
-message AddRoleRequest {
-  string identity_id = 1;
-  string role_id = 2;
-  string resource_type = 3;
-  string resource_id = 4;
-  string organization_id = 5;
+message ResourceFilter {
+  string resource_type = 2;
+  string resource_id = 3;
 }
 
-message AddRoleResponse {
+message AddRoleRequest {
   Authorization authorization = 1;
 }
 
+message AddRoleResponse {}
```

Comment:
wasn't sure why `AddRoleResponse` needed to return the added authorization, so I removed it. lmk if this should be reinstated

Reply:
Fine with it. I think we did this because is pretty standard to return the just created data as a response in our APIs. Not sure if we actually use this on the frontend, but will need to remove the logic on the backend to return just an empty response.

Reply:
ah if it's standard, should we leave it in?

Reply:
actually, going to delete it because the other `createX` cases seem to do some transformation on the request data